tests/issues/issue-2101.php

```diff
@@ -0,0 +1,3359 @@
+            'Device_Pointing_Method' => 'touchscreen',
+            'Device_Code_Name' => 'iPhone',
+            'Device_Brand_Name' => 'Apple',
+            'RenderingEngine_Name' => 'WebKit',
+            'RenderingEngine_Version' => 'unknown',
+            'RenderingEngine_Maker' => 'Apple Inc',
+        ],
+        'lite' => true,
+        'standard' => false,
+        'full' => true,
+    ],
+    'issue-2101-BF (standard)' => [
+        'ua' => 'Mozilla/5.0 (iPhone; CPU iPhone OS 12_4 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.1.2 Mobile/15E148 Safari/604.1',
+        'properties' => [
+            'Comment' => 'Mobile Safari 12.1',
+            'Browser' => 'Safari',
+            'Browser_Maker' => 'Apple Inc',
+            'Version' => '12.1',
+            'Platform' => 'iOS',
+            'Device_Type' => 'Mobile Device',
+            'Device_Pointing_Method' => 'touchscreen',
+        ],
+        'lite' => false,
+        'standard' => true,
+        'full' => false,
+    ],
+    'issue-2101-BG' => [
+        'ua' => 'Mozilla/5.0 (Linux; Android 9) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Web Explorer/2.6.1 Chrome/74.0.3729.157 Mobile Safari/537.36',
+        'properties' => [
+            'Comment' => 'Web Explorer',
+            'Browser' => 'Web Explorer',
+            'Browser_Type' => 'Browser',
+            'Browser_Bits' => '32',
+            'Browser_Maker' => 'Kawe Apps',
+            'Browser_Modus' => 'unknown',
+            'Version' => '0.0',
+            'Platform' => 'Android',
+            'Platform_Version' => 'unknown',
+            'Platform_Description' => 'Android OS',
+            'Platform_Bits' => '32',
+            'Platform_Maker' => 'Google Inc',
+            'Alpha' => false,
+            'Beta' => false,
+            'Frames' => true,
+            'IFrames' => true,
+            'Tables' => true,
+            'Cookies' => true,
+            'JavaScript' => true,
+            'VBScript' => false,
+            'JavaApplets' => false,
+            'isSyndicationReader' => false,
+            'isFake' => false,
+            'isAnonymized' => false,
+            'isModified' => false,
+            'CssVersion' => '3',
+            'Device_Name' => 'general Mobile Phone',
+            'Device_Maker' => 'unknown',
+            'Device_Type' => 'Mobile Phone',
+            'Device_Pointing_Method' => 'touchscreen',
+            'Device_Code_Name' => 'general Mobile Phone',
+            'Device_Brand_Name' => 'unknown',
+            'RenderingEngine_Name' => 'Blink',
+            'RenderingEngine_Version' => 'unknown',
+            'RenderingEngine_Maker' => 'Google Inc',
+        ],
+        'lite' => false,
+        'standard' => true,
+        'full' => true,
+    ],
+    'issue-2101-BH' => [
+        'ua' => 'Mozilla/5.0 (Linux; Android 9.0; SAMSUNG SM-G955F Build/PPR1.180610.011) AppleWebKit/537.36 (KHTML, like Gecko) SamsungBrowser/6.2 Chrome/56.0.2924.87 Mobile Safari/537.36',
```

Comment:
I think we should just remove this test.

Reply:
removed